common/v2/components/NetworkNodeDropdown.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useCallback, useContext, FC, useState, useEffect } from 'react';
+import { OptionComponentProps } from 'react-select';
+import styled from 'styled-components';
+import isEmpty from 'lodash/isEmpty';
+import isFunction from 'lodash/isFunction';
+
+import { NetworkContext, NetworkUtils } from 'v2/services/Store';
+import { CustomNodeConfig, NetworkId, NodeOptions } from 'v2/types';
+import { Typography, Dropdown } from 'v2/components/index';
+import { translateRaw } from 'v2/translations';
+import { SPACING, COLORS } from 'v2/theme';
+
+import addIcon from 'assets/images/icn-add.svg';
+import editIcon from 'assets/images/icn-edit.svg';
+
+const SContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding: 12px;
+
+  &:hover {
+    background-color: var(--color-gray-lighter);
```

Comment:
please use TS styling constants with styled components

Reply:
This hover thing can actually be deleted, hover state is now handled in the `Dropdown` component.